### PR TITLE
Add 2 docs

### PR DESCRIPTION
**Pull request by TomShawn**
Signed-off-by: TomShawn <41534398+TomShawn@users.noreply.github.com>

This PR adds create-chaos-mesh-workflow.md and simulate-kernel-chaos-on-kubernetes.md.

Translated from https://github.com/chaos-mesh/website-zh/pull/22 and https://github.com/chaos-mesh/website-zh/pull/36.

### CODE REVIEW

File: docs/simulate-kernel-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -1,6 +1,140 @@
...
+title: Simulate Linux Kernel Faults
+sidebar_label: Simulate Linux Kernel Faults
 ---
 
-## TODO
+This document describes how to use KernelChaos to simulate Linux kernel faults. This feature injects I/O-based or memory-based faults into the specified kernel paths using [BPF](https://lore.kernel.org/lkml/20171213180356.hsuhzoa7s4ngro2r@destiny/T/).
+
+Although you can set the injection target of KernelChaos to one or several Pods, the performance of other Pods on the host will be affected, because all Pods share the same kernel.
+
+::: warning
```
Comment (qiancai) [inline docs/simulate-kernel-chaos-on-kubernetes.md:10]:
```suggestion
:::warning
```

**Review by cwen0 — APPROVED** [review]
LGTM

File: docs/create-chaos-mesh-workflow.md [hunk 2]
```diff
@@ -1,6 +1,202 @@
...
+When you use Chaos Mesh to simulate real system faults, continuous validation is always a need. You might want to build a series of faults on the Chaos Mesh platform, instead of performing individual Chaos injections.
+
+To meet this need, Chaos Mesh provided Chaos Mesh Workflow, a built-in workflow engine. Using this engine, you can run different Chaos experiments in serial or parallel to simulate production-level errors.
+
+Currently, Chaos Mesh Workflow supports the following features:
+
+- Serial Orchestration
+- Parallel Orchestration
+- Customized tasks (work in progress)
+- Conditional branch (work in progress)
```
Comment (STRRL) [inline docs/create-chaos-mesh-workflow.md:17]:
```suggestion
- Customized tasks
- Conditional branch
```

这两个目前已经完成了，中文文档中也提了对应的 PR 删掉这个 WIP.  https://github.com/chaos-mesh/website-zh/pull/57

File: docs/create-chaos-mesh-workflow.md [hunk 3]
```diff
@@ -1,6 +1,202 @@
...
+| children            | []string            | Declares the subtasks under this template. You need to configure this field when the type is `Serial` or `Parallel`.                                                                                                        | None          | No       | ["any-chaos-1", "another-serial-2", "any-shcedue"] |
+| task                | Task                | Configures the customized task. You need to configure this field when the type is `Task`. See the [Task field description](#task-field-description) for details.                                                            | None          | No       |                                                    |
+| conditionalBranches | []ConditionalBranch | Configures the conditional branch after the task is customized. You need to configure this field when the type is `Task`. See the [Conditional branch field description](#conditionalbranch-field-description) for details. | None          | No       |                                                    |
```
Comment (STRRL) [inline docs/create-chaos-mesh-workflow.md:151]:
```suggestion
| conditionalBranches | []ConditionalBranch | Configures the conditional branch which executes after customized task. You need to configure this field when the type is `Task`. See the [Conditional branch field description](#conditionalbranch-field-description) for details. | None          | No       |                                                    |
```

File: docs/create-chaos-mesh-workflow.md [hunk 4]
```diff
@@ -1,6 +1,202 @@
...
+| ---------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | -------- | ------------- |
+| target     | string | The name of the template to be executed by the current conditional branch.                                                                                                                                                                                             | None          | Yes      | another-chaos |
+| expression | string | The type is a boolean expression. When a customized task is completed and the expression value is true, the current condition branch is executed. When this value is not set, the conditional branch will be executed directly after the customized task is completed. | None          | No       | exitCode == 0 |
+
+Currently, two context variables are provided in `expression`:
+
+- `exitCode` means the exit code for a customized task.
+- `stdout` indicates the standard output for a custom task.
```
Comment (STRRL) [inline docs/create-chaos-mesh-workflow.md:188]:
```suggestion
- `exitCode` means the exit code for a customized task.
- `stdout` indicates the standard output for a customized task.
```

**Pull request merged**